openapi.yaml

```diff
@@ -138,14 +144,35 @@ paths:
 components:
   schemas:
     AsyncAPIDocument:
-      description: AsyncAPI document in JSON or YAML
+      description: AsyncAPI document in JSON or YAML.
       oneOf:
         - type: string # can be in YAML format
         - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.0.0.json
         - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.1.0.json
         - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.2.0.json
         - $ref: https://raw.githubusercontent.com/asyncapi/spec-json-schemas/master/schemas/2.3.0.json
-    Template:
+    AsyncAPIDocuments:
+      type: array
+      description: AsyncAPI documents in JSON or YAML.
+      minItems: 1
+      items:
+        $ref: '#/components/schemas/AsyncAPIDocument'
+    SpecVersions:
+      type: string
+      description: Valid specification versions for the AsyncAPI document.
```

Comment:
All the provided documents should stick to the same version? 
Related to this, would you mind refreshing why this should be provided instead of "guessing" from the document?

Reply:
This is the "schema" used to determine what versions we currently have supported. This is used in a convert request where we need to validate that the user has provided a valid value - one of the possible values.